src/cookieSync.js

```diff
@@ -93,7 +119,51 @@ function ajax(url, callback, data, options = {}) {
   }
 }
 
-// Send empty data to receive cookie sync status for all prebid server adapters.
-// In next phase we will read placement id's from query param and will only get cookie sync status of bidders participating in auction
-var data = '{}';
-ajax(ENDPOINT, process, data);
\ No newline at end of file
+function getUrlParam(paramName) {
+  if (paramName && paramName.length > 0) {
+    var sPageURL = window.location.search.substring(1);
+    if (sPageURL) {
+      var sURLVariables = sPageURL.split('&');
+      for (var i = 0; i < sURLVariables.length; i++) {
+        var sParameterName = sURLVariables[i].split('=');
+        if (sParameterName && sParameterName.length == 2 && sParameterName[0] == paramName) {
+          return sParameterName[1];
+        }
+      }
+    }
+  }
+}
+
+function getBidders() {
+  var bidders = getUrlParam("bidders");
+  if (bidders && bidders.length > 0) {
+    var bidderArray = bidders.split(",");
+    if (bidderArray && bidderArray.length > 0) {
+      return bidderArray;
+    } else {
+      return [
+        "appnexus",
+        "audienceNetwork",
+        "pubmatic",
+        "rubicon",
+        "pulsepoint",
+        "indexExchange",
+        "lifestreet"
+      ];
+    }
+  }
+}
+
+
+var data = JSON.stringify({
+  "uuid": "pubmatic" + (Math.random() * 10000).toFixed(4),
```

Comment:
what is the purpose of sending current value of uuid? what if we set it to empty? is it just a cache-buster?

Reply:
after discussion with jaydeep removed it